common/annotations/en.xml

```diff
@@ -3362,23 +3362,23 @@ annotations.
 		<annotation cp="🗿" type="tts">moai</annotation>
 		<annotation cp="🪧">demonstration | picket | placard | protest | sign</annotation>
 		<annotation cp="🪧" type="tts">placard</annotation>
-		<annotation cp="🏧">atm | ATM sign | automated | bank | teller</annotation>
+		<annotation cp="🏧">ATM | ATM sign | automated | bank | teller</annotation>
 		<annotation cp="🏧" type="tts">ATM sign</annotation>
 		<annotation cp="🚮">litter | litter bin | litter in bin sign</annotation>
 		<annotation cp="🚮" type="tts">litter in bin sign</annotation>
 		<annotation cp="🚰">drinking | potable | water</annotation>
 		<annotation cp="🚰" type="tts">potable water</annotation>
 		<annotation cp="♿">access | wheelchair symbol</annotation>
 		<annotation cp="♿" type="tts">wheelchair symbol</annotation>
-		<annotation cp="🚹">lavatory | man | men’s room | restroom | wc</annotation>
+		<annotation cp="🚹">lavatory | man | men’s room | restroom | WC</annotation>
 		<annotation cp="🚹" type="tts">men’s room</annotation>
-		<annotation cp="🚺">lavatory | restroom | wc | woman | women’s room</annotation>
+		<annotation cp="🚺">lavatory | restroom | WC | woman | women’s room</annotation>
```

Comment:
'bathroom' is commonly used in en-US; why is 'bathroom' and/or toilet not listed which are very common in en?

Reply:
Let me ping the ESC on that; the question is whether to add 'bathroom' and 'toilet' to the following.
`
		<annotation cp="🚹">lavatory | man | men’s room | restroom | WC</annotation>
		<annotation cp="🚺">lavatory | restroom | WC | woman | women’s room</annotation>
		<annotation cp="🚻">lavatory | restroom | WC</annotation>
		<annotation cp="🚾">closet | lavatory | restroom | water | WC</annotation>
`
I'd like to go ahead with this for now, and if they get back to me by tomorrow, then add in separate PR.

Reply:
Makes sense :) I just found that other than 'restroom' all the annotations sound super foreign to me. I almost expect to see 'Loo' and 'Head' in the list.